TestAssets/TestProjects/AppWithDependencyOnMalformedPackage/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "BadPackage": "1.0.0"
```

Comment:
I think if you must check it in, at least put in the nuspec or some way to regenerate the package if need be